etl/multidim.py

```diff
@@ -162,3 +167,74 @@ def fetch_variables_from_table(table: str, engine: Engine) -> pd.DataFrame:
     df_dims = pd.DataFrame(dims, index=df.index)
 
     return df.join(df_dims)
+
+
+def generate_views_for_dimensions(
+    dimensions, tables, dimensions_order_in_slug=None, additional_fields=None, warn_on_missing_combinations=True
```

Comment:
Maybe add a docstring to clarify what each argument is doing?

From the API alone it is not clear to me what, e.g., `additional_fields` is doing, I had to read the function's code.

My understanding is that you are using it as top-level configuration (i.e. applies to _all_ indicators)?